packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -94,8 +94,8 @@ class ScaleUpdateDetails {
        assert(rotation != null),
        localFocalPoint = localFocalPoint ?? focalPoint;
 
-  /// The amount the pointer has moved in the coordinate space of the event
-  /// receiver since the previous update.
+  /// The amount the gesture's focal point has moved in the coordinate space of
+  /// the event receiver since the previous update.
   ///
   /// Defaults to zero if not specified in the constructor.
   final Offset delta;
```

Comment:
Maybe we should use a more self-explanatory name instead of `delta`?

Reply:
I'm trying to mirror the delta parameter in DragUpdateDetails, but it's true it's not exactly the same thing.  Maybe `focalPointDelta`?

https://github.com/flutter/flutter/blob/529a599ce64bb3aafe7999ac9364be86ad29afb1/packages/flutter/lib/src/gestures/drag_details.dart#L156-L164

Reply:
Yeah that sounds good! I was a bit confused about what `delta` was since there are at least 2 pointers involved in a scale gesture.